dependencies.lock

```diff
@@ -1,750 +0,0 @@
-{
```

Comment:
we need that file. without it, we have either no control over our dependencies (when using wildcards in the build.gradle like `1.7.+`) or we need to lock the dependencies in the build.gradle to exact versions and need to watch for updates ourselves. `gradlew lock` should re-create this file and then you should add and commit the new version.